apps/cowswap-frontend/src/modules/tradeQuote/hooks/useProcessUnsupportedTokenError.ts

```diff
@@ -4,16 +4,17 @@ import { getQuoteUnsupportedToken } from '@cowprotocol/common-utils'
 import { useAddUnsupportedToken } from '@cowprotocol/tokens'
 
 import QuoteApiError from 'api/cowProtocol/errors/QuoteError'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
 
 export function useProcessUnsupportedTokenError() {
   const addGpUnsupportedToken = useAddUnsupportedToken()
 
   return useCallback(
-    (error: QuoteApiError, quoteParams: { sellToken: string; buyToken: string }) => {
+    (error: QuoteApiError, quoteParams: { chainId: SupportedChainId; sellToken: string; buyToken: string }) => {
       const unsupportedTokenAddress = getQuoteUnsupportedToken(error, quoteParams)
 
       if (unsupportedTokenAddress) {
-        addGpUnsupportedToken(unsupportedTokenAddress)
+        addGpUnsupportedToken(quoteParams.chainId, unsupportedTokenAddress)
```

Comment:
we need to pass the chainId used for the quote, this is how we make sure the error matches the chain